src/applications/claims-status/containers/AdditionalEvidencePage.jsx

```diff
@@ -71,16 +87,15 @@ class AdditionalEvidencePage extends React.Component {
     } else {
       const claim = this.props.claim;
       const filesPath = `your-claims/${claim.id}/files`;
-      const claimsPath = `your-claims${claim.attributes.open ? '' : '/closed'}`;
+      // const claimsPath = `your-claims${claim.attributes.open ? '' : '/closed'}`;
```

Comment:
Similar to above, is this change intentional? If so, should the comment be removed?